docs/_posts/2018-11-19-android.md

```diff
@@ -382,6 +382,38 @@ Configuration above works out of the box for allure-kotlin 2.3.0+.
 
 Additional configuration parameters include **pathRoot** which has two options: `EXTERNAL_STORAGE` that is usually the `/sdcard/` on most of the devices and `APP_DATA` which is usually `/data/data/$appPackage/`.
 Besides the expected path root you might need to provide the **relativeResultsDirectory**: this is the relative path to `pathRoot`. The default path for allure-kotlin as of 2.3.0 is `/data/data/$appPackage/allure-results`. 
+
+{% tabs allure-kotlin %}
```

Comment:
The tab id has to be unique across the document. `allure-kotlin` is already used above